aspnetcore/mvc/views/tag-helpers/built-in/distributed-cache-tag-helper.md

```diff
@@ -44,7 +54,7 @@ Usage Example:
 
 ## Distributed Cache Tag Helper IDistributedCache implementations
 
-There are two implementations of `IDistributedCache` built in to ASP.NET Core. One is based on SQL Server and the other is based on Redis. Details of these implementations can be found at <xref:performance/caching/distributed>. Both implementations involve setting an instance of `IDistributedCache` in ASP.NET Core's *Startup.cs*.
+There are two implementations of <xref:Microsoft.Extensions.Caching.Distributed.IDistributedCache> built in to ASP.NET Core. One is based on SQL Server, and the other is based on Redis. Details of these implementations can be found at <xref:performance/caching/distributed>. Both implementations involve setting an instance of `IDistributedCache` in ASP.NET Core's *Startup.cs*.
```

Comment:
Both implementations involve setting an instance of `IDistributedCache` in `Startup`.